Sources/PointFree/Home.swift

```diff
@@ -21,25 +21,24 @@ let secretHomeMiddleware: (Conn<StatusLineOpen, Tuple3<Database.User?, Stripe.Su
           currentSubscriptionStatus: currentSubscriptionStatus,
           currentUser: currentUser,
           data: (currentUser, currentSubscriptionStatus),
-          extraStyles: pricingExtraStyles,
+          extraStyles: pricingExtraStyles <> blueGradientStyles <> reflectStyles,
```

Comment:
these extra styles could prob be moved to our design systems lib, but just leaving in this file for now